client/v3/balancer/balancer.go

```diff
@@ -138,6 +138,18 @@ type baseBalancer struct {
 	picker picker.Picker
 }
 
+// UpdateClientConnState implements "grpc/balancer.Balancer" interface.
+func (bb *baseBalancer) UpdateClientConnState(ccs balancer.ClientConnState) error {
+	bb.HandleResolvedAddrs(ccs.ResolverState.Addresses, nil)
+	// TODO: This **should** incorporate the lines that warn in `HandleResolvedAddrs`.
```

Comment:
E.g. here: https://github.com/etcd-io/etcd/blob/5dcd459ae9c7948f5620002f5b0bb9cf0b8f1502/client/v3/balancer/balancer.go#L145

My goal here was to